.taskcluster.yml

```diff
@@ -7,10 +7,10 @@ tasks:
     - $let:
           # XXX Set to `true` for private repos
           privateRepo: false
-          # XXX use "privileged", "system", or "mozillaonline-privileged" to
-          #     enable signing on push/PR. Note, this only works for public
-          #     repos for now. Release builds off shipit will get signing for
-          #     both public and private repos.
+          # Use `system` for system add-on,
+          # `hosted-privileged` for AMO or self-hosted privileged add-on,
```

Comment:
We'll probably want `hosted` to agree with whatever we name it in `xpi-manifest`.